bin/pos-cli-data-export.js

```diff
@@ -47,25 +49,44 @@ program
     const exportInternalIds = params.exportInternalIds;
     const authData = fetchAuthData(environment, program);
     gateway = new Gateway(authData);
+
+    const isZipFile = path.extname(filename) === '.zip';
```

Comment:
I'm not sure about magical compression detection. Usually output format is controlled explicitly by parameter.